src/coreclr/vm/jitinterface.cpp

```diff
@@ -13102,11 +13085,7 @@ PCODE UnsafeJitFunction(PrepareCodeConfig* config,
             // Note: if we're only importing (ie, verifying/
```

Comment:
This comment does not make sense. (It did not make sense even before this change.)